sklearn/model_selection/_validation.py

```diff
@@ -1863,8 +2076,11 @@ def _incremental_fit_estimator(
 
         start_score = time.time()
 
-        test_scores.append(_score(estimator, X_test, y_test, scorer, error_score))
-        train_scores.append(_score(estimator, X_train, y_train, scorer, error_score))
+        # TODO(SLEP6): support score params in the following two calls
+        test_scores.append(_score(estimator, X_test, y_test, scorer, None, error_score))
+        train_scores.append(
+            _score(estimator, X_train, y_train, scorer, None, error_score)
+        )
```

Comment:
Can we use keyword arguments as least for the last two arguments because it makes the code more readable that way when we know what parameter is being assigned None.
```suggestion
        test_scores.append(
            _score(estimator, X_test, y_test, scorer, score_params=None, error_score=error_score)
        )
        train_scores.append(
            _score(estimator, X_train, y_train, scorer, score_params-None, error_score=error_score)
        )
```